src/group_theory/perm/basic.lean

```diff
@@ -267,6 +288,30 @@ instance perm_unique {n : Type*} [unique n] : unique (equiv.perm n) :=
 
 @[simp] lemma default_perm {n : Type*} : default (equiv.perm n) = 1 := rfl
 
+variables (e : perm α) (ι : α ↪ β)
+
+open_locale classical
+
+/-- Extend a permutation along an embedding -/
+noncomputable def of_embedding : perm β :=
+extend_domain e (of_injective ι.1 ι.2)
```

Comment:
Does `equiv.perm.via_embedding` work here? If not, can you adjust the name and docstring to compare the two?

Reply:
`equiv.perm.via_embedding` is computable, by assuming `fintype`. I've updated the docstring. What name do you suggest?